kfs-core/src/main/java/edu/arizona/kfs/gl/batch/service/impl/BudgetAdjustmentCashTransferServiceImpl.java

```diff
@@ -466,21 +484,16 @@ public int buildTransactionsForTransferOfFundsGeneration(String documentNumber,
                        reportOriginEntryGenerated++;                                                              
                    }
                    catch (IOException ioe) {
-                       LOG.error("generateBudgetAdjustmentCashTransferTransactions Stopped: " + ioe.getMessage());
-                       throw new RuntimeException("generateBudgetAdjustmentCashTransferTransactions Stopped: " + ioe.getMessage(), ioe);
+                       LOG.error("generateBudgetAdjustmentCashTransferTransactions Stopped: ", ioe);
+                       throw new RuntimeException(ioe);
                    }                   
                    
                 } //matching transaction found 
             }
             catch (RuntimeException re) {
-                LOG.error("generateBudgetAdjustmentCashTransferTransactions Stopped: " + re.getMessage());
-                throw new RuntimeException("generateBudgetAdjustmentCashTransferTransactions Stopped: " + re.getMessage(), re);
-            }
-            catch (Exception ex) {
```

Comment:
When we ran batch in KFS 3.0, did we ever encounter exceptions that are caught here and written to the report? Like maybe string parsing exceptions? Or some some info missing that makes impossible to calculate adjustment for that particular line?